### ISSUE TITLE
Invalid Security Certificate issue experienced by Idaho-Montana user

### ISSUE THREAD

**Issue by OhMcGoo** [redcross/smoke-alarm-portal#222]
Hi Jim,
One of my volunteers alerted me to difficulty accessing getasmokealarm.org because of the website’s invalid security certificate.  Tests I have completed on Chrome and Safari confirm her report.  Is this an issue your team is aware of and working to resolve?
Best,
Ted
 
[edited to remove personal info by @cecilia-donnelly]

**Comment by cecilia-donnelly** [comment]
@OhMcGoo: We're back in business!  @kfogel will follow up.

**Comment by kfogel** [comment]
I will indeed follow up, leading with an apology!  Sorry; process snafu here.  Jim, I'll send you a followup email.  Basically: we switched cert provider to LetsEncrypt on most of the sites we manage -- but we didn't do it for this one, because we'd already purchased the renewal cert anyway, which I then neglected to install at the right time.  (And, of course, this kind of failure is AFAIK invisible to the monitoring service we're now using, since the site is still up and responding -- though I'm going to look and see if we can tweak that.)
